engine/src/test/java/org/operaton/bpm/engine/test/api/filter/FilterTaskQueryTest.java

```diff
@@ -63,7 +74,20 @@
 /**
  * @author Sebastian Menski
  */
-public class FilterTaskQueryTest extends PluggableProcessEngineTest {
+public class FilterTaskQueryTest {
+  
+  @RegisterExtension
+  protected static ProcessEngineExtension engine = new ProcessEngineExtension();
```

Comment:
The ProcessEngineExtension should always be created via the builder when instantiating for ' @RegisterExtension`. Please verify other usages of the default constructor.